Formula/gotify-cli.rb

```diff
@@ -0,0 +1,25 @@
+class GotifyCli < Formula
+  desc "Command-line interface for pushing messages to gotify/server"
+  homepage "https://github.com/gotify/cli"
+  url "https://github.com/gotify/cli/archive/refs/tags/v2.2.0.tar.gz"
+  sha256 "d4e6fbc087c6cc4b320b3fcae4c28d65d514b7e55c95e4c0e764abb3a6c94064"
+  license "MIT"
+  head "https://github.com/gotify/cli.git", branch: "master"
+
+  livecheck do
+    url "https://github.com/gotify/cli/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-X main.Version=#{version}
+                                       -X main.BuildDate=2021-12-09-12:45:29
+                                       -X main.Commit=c8bb424d497f42661e061e93f9e0068852b14975")
```

Comment:
I was not sure how to insert this dynamically, can someone help me with this?

Reply:
You can't if you're using a tarball for the source. If this isn't strictly required I'd leave it out.

Reply:
It's used in the version output. My idea was to just use `latest`:

```
Version:   2.2.0
Commit:    latest
BuildDate: 2020-07-11T13:50:59Z
```

Reply:
I'd go for N/A. It might not always be the latest after all.

Reply:
Ok, done, thanks :)